test/companies/pending_changes_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule Companies.PendingChangesTest do
+  use Companies.DataCase
+
+  import Companies.Factory
+  import ExUnit.CaptureLog
+
+  alias Companies.{Companies, PendingChanges}
+
+  describe "all/0" do
+    test "retrieve all pending changes" do
+      insert(:pending_change)
+      insert(:pending_change, %{approved: true})
+      assert 1 == length(PendingChanges.all())
+    end
+  end
+
+  describe "approve/2" do
+    test "approves a pending create change" do
+      %{total_entries: pre_count} = Companies.all()
+
+      %{id: id} = insert(:pending_change)
+      assert {:ok, %{approved: true}} = PendingChanges.approve(id, true)
+      assert 0 == length(PendingChanges.all())
+
+      %{total_entries: post_count} = Companies.all()
+
+      assert pre_count < post_count
+    end
+
+    test "approves a pending update change" do
+      %{id: company_id} = insert(:company)
+      %{id: id} = insert(:pending_change, %{action: "update", changes: %{id: company_id, name: "updated"}})
+
+      assert {:ok, %{approved: true}} = PendingChanges.approve(id, true)
+      assert %{entries: [%{name: "updated"}], total_entries: 1} = Companies.all()
+    end
+
+    test "approves a pending delete change" do
+      %{id: company_id} = insert(:company)
+      %{id: id} = insert(:pending_change, %{action: "delete", changes: %{id: company_id}})
+
+      assert {:ok, %{approved: true}} = PendingChanges.approve(id, true)
+      assert %{total_entries: 0} = Companies.all()
+    end
+
+    test "rejects a pending change" do
+      %{total_entries: pre_count} = Companies.all()
+
+      %{id: id} = insert(:pending_change)
+      assert {:ok, %{approved: false}} = PendingChanges.approve(id, false)
+      assert 0 == length(PendingChanges.all())
```

Comment:
use `Enum.empty?/1` intead of `0 == length(PendingChanges.all())`

Reply:
Done!